xc/device.ml

```diff
@@ -743,8 +769,8 @@ let release (task: Xenops_task.t) ~xs (x: device) =
 		Hotplug.run_hotplug_script x [ "remove" ];
 		Hotplug.run_hotplug_script tap [ "remove" ];
 	end;
-	Hotplug.release task ~xs x
-
+	Hotplug.release task ~xs x;
+	remove_extra_xenserver_keys ~xs x
```

Comment:
The code for `remove_extra_xenserver_keys` can be moved into `Hotplug.release`, since it is doing the same kind of thing.